annotations/component-annotations/src/main/java/io/dekorate/component/annotation/ComponentApplication.java

```diff
@@ -44,4 +44,6 @@
   Env[] envs() default {};
 
   String buildType() default "s2i";
+
+  String remote() default "";
```

Comment:
Default value should be origin.

Reply:
Default value is 'origin' as when Scm info is built origin is setted:
`String url = Git.getRemoteUrl(path, **Git.ORIGIN**).map(u -> u.replace(Git.GITHUB_SSH, Git.GITHUB_HTTPS)).orElse(null);`
If it's expressed at annotation level, in the handler it will never be empty and hard to know if it should be overrided. Am I wrong?

Reply:
Not sure why that's a problem. Instead of checking for string being empty you can check for string being equal to "origin".

It's no biggie, but its always better if defaults are clearly expressed. Easier for the user to understand what's going on...

Reply:
Yes, true, I didn't see it that way but yes, it's better like this :-)